roles/neutron-common/templates/etc/neutron/plugins/ml2/ml2_plugin.ini

```diff
@@ -19,6 +19,9 @@ tenant_network_types = {{ neutron.tenant_network_type }}
 {% endif %}
 mechanism_drivers = {{ mechanism_drivers|join(',') }}
 
+{% set extension_drivers = ['port_security'] %}
+extension_drivers = {{ extension_drivers|join(',') }}
```

Comment:
Yeah, I'm not a fan of how the mechanism bit has been done.  I think it's leftover from a larger logic set that no longer exists.

I'd prefer to have a role default for what extensions to enable. We can define port_security as a role default, and envs can add more or remove.